pkg/container/docker_run.go

```diff
@@ -232,6 +229,21 @@ func GetDockerClient(ctx context.Context) (*client.Client, error) {
 	return cli, err
 }
 
+func GetHostInfo(ctx context.Context) (info types.Info, err error) {
+	var cli *client.Client
+	cli, err = GetDockerClient(ctx)
+	if err != nil {
+		return info, err
+	}
```

Comment:
Please close the docker connection, after using it.
```suggestion
	cli, err = GetDockerClient(ctx)
	if err != nil {
		return info, err
	}
	defer cli.Close()
```